db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ ] [REQUIRED] Kindly add  5 greetings to your seed file so that a reviewer can test them as stated by the project requirement.

Reply:
Fixed:
![image](https://github.com/Ashnaali3255/Rails-React-Project/assets/116655378/d46978a0-ebe6-47e3-8c62-57d992095037)